javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,78 @@
+//Start of A future age calculator
+console.log("A future age calculator");
+const yearOfBirth = 1990;
+const yearFuture = 2040;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+//End of future age calculator
+
+
+
+//A dog age  calculator//
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2030;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears) {
+  console.log(`Your dog will be ${dogYear * 7} dog years old in ${dogYearFuture}`);
+} else {
+  console.log(`Your dog will be ${dogYear} human years old in ${dogYearFuture}`);
+}
+//end of dog age calculator
+
+
+
+
+
+// house price estimater
+//peter house price calculation
+let width = 8;
+let depth = 10;
+let hight = 10;
+let gardenSize = 100;
+let VolumeInMeters = width*depth*hight;
+let peterestimatedhousePrice = VolumeInMeters * 2.5 *1000 + gardenSize *300;
+let peteractualhouseprice = 2500000;
+if (peteractualhouseprice > peterestimatedhousePrice){
+    console.log("peter paid little");
+}
+else{
+    console.log("peter paid little");
+}
+
+//julia house price
+
+let wide = 5;
+let deep = 8;
+let high = 11;
+let gardensize = 70;
+let volumeInMeters = wide*deep*high;
+let juliasstimatedhousePrice = volumeInMeters * 2.5 *1000 + gardensize *300;
+let juliasactualhouseprice = 1000000;
+if (juliasactualhouseprice < juliastimatedhousePrice){
+    console.log("julia paid little");
+}
+else{
+    console.log("julia paid little");
+}
+
+
+
+
+
+
+
+
+
+
+
+//startup name generator
+console.log("start up name")
+let  firstWords = ["easy","super","lazend","super","good","advik","some","tasty","sam","next",];
+let  secondWords =["small","bigshop","milagrow","smile","dwarka","haveli","nice","spice","create","future"];
+
+const randomNumber = math.floor(Math.random() * 10);
+
+const startupName = firstWords[randomNumber] + secondWords [randomNumber];
+console.log('the startup is ${satrtupname} and conatains ${startupName.lengh} charecters');
```

Comment:
There are some syntax errors here that are making the code not work:
1-  Fix `math.floor` to be `Math.floor`;

2- There is a white space between `secondWords [randomNumber]`, remove the white space;

3 - The variable `startupName`, is spelled wrong in the console.log and therefore, an error is thrown ;

4 - You are trying to use a template string in the `console.log`, but instead of using backticks ``, you are using quotation marks `''`,  therefore the console will log only a string like this:
<img width="605" alt="Screenshot 2023-03-04 at 10 10 42" src="https://user-images.githubusercontent.com/99715716/222888891-df46d22d-1e16-4ee8-a9d6-a7bd7a3b6d32.png"> 

After fixing these mistakes, the code should work fine😉

Reply:
thank you